CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# TiDB Operator v1.0.3 Release Notes
+
+## v1.0.3 What's New
+
+### Action Required
+
+This release adds the `timezone` support for [all charts](https://github.com/pingcap/tidb-operator/tree/master/charts).
+
+For existing tidb clusters. If the `timezone` in `tidb-cluster/values.yaml` has been customized to other timezones instead of the default `UTC`, then upgrading tidb-operator will trigger a rolling update for the related pods.
```

Comment:
```suggestion
For existing TiDB clusters. If the `timezone` in `tidb-cluster/values.yaml` has been customized to other timezones instead of the default `UTC`, then upgrading tidb-operator will trigger a rolling update for the related pods.
```